operators/view_history.py

```diff
@@ -112,8 +112,8 @@ def execute(self, context):
             self.report({"ERROR"}, "No valid selection to export.")
             return {"FINISHED"}
         with open(self.filepath, 'w', encoding='utf-8') as target:
-            args = selection.generate_args()
-            args['seed'] = selection.seed
+            args = {key: getattr(selection, key) for key in DreamPrompt.__annotations__}
```

Comment:
`generate_args()` is now too purpose-built in preparing arguments for the backend. All we really need are the raw values to save and restore the DreamPrompt state.